ci/terraform/modules/endpoint-lambda/variables.tf

```diff
@@ -74,6 +74,15 @@ variable "extra_tags" {
   description = "Extra tags to apply to resources"
 }
 
+locals {
+  extra_tags = merge(
+    var.extra_tags,
+    {
+      Service = var.endpoint_name
+    }
+  )
+}
```

Comment:
Its not clear what is happening here, what is the difference between var.extra_tags and local.extra_tags in preceding changes.

Reply:
resources don't get 'Service' by default, so we add a value for 'Service' based on the endpoint name. Merging this one extra tag with `var.extra_tags` and then using the combination instead of the bare variable is the simplest way to keep both the specifically defined extra_tags and the additional service one.